doc/flatpak-install.xml

```diff
@@ -269,6 +269,23 @@
                 </para></listitem>
             </varlistentry>
 
+            <varlistentry>
+                <term><option>--include-sdk</option></term>
+
+                <listitem><para>
+                  For each app being installed, also installs the SDK that was used to build it.
```

Comment:
"Implies --or-update. Incompatible with --disable-deps"?

Reply:
I'm...not sure what `--disable-deps` is or why it's not compatible, it's not on the man page :sweat_smile:

Reply:
I must have meant `--no-deps`